### DISCUSSION TITLE
NET market price - naming misleading?

### DISCUSSION THREAD

**Discussion by riesi77** [mampfes/ha_epex_spot#136] [General]
I'm always confused when I read NET market price and that it includes surcharge and taxes (VAT). I would rather call it gross market price. Where does this "net" come from? Usually it means without VAT...

**Comment by mampfes** [comment]
This has been discussed multiple times already - please use the search function. What I plan to do is to remove the "raw" sensor and only provide one sensor which includes the configured surcharge / VAT.

**Reply by riesi77** [reply]
When searching for "net", I can not find any result answering my question. Can you please lead me to the discussion you mean?
